docs/bugs.json

```
@@ -1,4 +1,15 @@
 [
+    {
+        "uid": "SOL-2022-3",
+        "name": "DataLocationChange",
+        "summary": "It was possible to override internal and public function while changing the data location of the parameters or return variables from ``calldata`` to ``memory`` and vice-versa. This caused invalid code to be generated when calling such a function internally through virtual function calls.",
```

It sounds a bit weird in that order. I'd reverse that sentence:
```suggestion
        "summary": "It was possible to change the data location of the parameters or return variables from ``calldata`` to ``memory`` and vice-versa while overriding internal and public functions. This caused invalid code to be generated when calling such a function internally through virtual function calls.",
```